lib/world.rb

```diff
@@ -50,6 +51,25 @@ def debug_in_after_hook
       end
     end
 
+    def print_debugging_info(hos)
+      # Intention of method is to execute additional debug statement to analyze most trickier failures.
+      # Hack(below two lines) to ignore unwanted logger messages.
+      logger.write_custom_errmsg('')
+      logger.set_log_ignore(true)
+      logger.set_log_level('PLAIN')
+      res = admin.cli_exec(:get, resource: "nodes")
+      nodes = hos.exec_raw("echo '#{res[:response]}' | grep -E 'SchedulingDisabled|NotReady' | awk '(NR>1)'")
+      res = admin.cli_exec(:get, all_namespaces: true, resource: "co")
+      cos = hos.exec_raw("echo '#{res[:response]}' | grep -v '.True.*False.*False' | awk '(NR>1)'")
+      res = admin.cli_exec(:get, all_namespaces: true, resource: "pods")
+      pods = hos.exec_raw("echo '#{res[:response]}' | grep -Ev 'Running|Completed' | awk '(NR>1)'")
```

Comment:
I remember we had `_silent: true` options when executing commands. Maybe you can use this option instead of this custom log filtering? Maybe it was `silent: true` for `hos.exec_raw`. You can `grep` through the existing codebase to see.

btw perhaps you meant `host` instead of `hos`?

Reply:
I did grep search in repo about `silent`, but couldn't see any related implementation. 
To avoid confusion, I have changed the variable name from `hos` to `host` now.

Reply:
ops, sorry, it is `quiet: true` for host commands and `_quiet: true` for oc cli commands.

Reply:
Wow, this is very useful and reduces the additional hacks. Thanks for the suggestion. Updated now.